app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -746,11 +746,13 @@ class Settings(private val appContext: Context) : PreferencesHolder {
             camera = getSitePermissionsPhoneFeatureAction(PhoneFeature.CAMERA),
             autoplayAudible = getSitePermissionsPhoneFeatureAutoplayAction(PhoneFeature.AUTOPLAY_AUDIBLE),
             autoplayInaudible = getSitePermissionsPhoneFeatureAutoplayAction(PhoneFeature.AUTOPLAY_INAUDIBLE),
-            persistentStorage = getSitePermissionsPhoneFeatureAction(PhoneFeature.PERSISTENT_STORAGE)
+            persistentStorage = getSitePermissionsPhoneFeatureAction(PhoneFeature.PERSISTENT_STORAGE),
+            mediaKeySystemAccess = getSitePermissionsPhoneFeatureAction(PhoneFeature.MEDIA_KEY_SYSTEM_ACCESS)
         )
     }
 
     fun setSitePermissionSettingListener(lifecycleOwner: LifecycleOwner, listener: () -> Unit) {
+        // TODO: Should this list also contain PERSISTENT_STORAGE / MEDIA_KEY_SYSTEM_ACCESS?
```

Comment:
Yes, it looks like I missed it in the previous pr, thanks for catching it.

Reply:
It will be great if we could add a test for verifying it and avoid future regressions

Reply:
Cool, thanks! Do you know how I could test this? I guess something like dynamically changing a setting while a session is open or something, but I'm not sure how to best do that :)

Reply:
Yes, after taking a closer looks seem a bit over complicated to test 😄 , I think it doesn't worth the effort. Please feel free to skip it 👍🏽